bigchaindb/lib.py

```diff
@@ -483,6 +480,14 @@ def migrate_abci_chain(self):
 
         self.store_abci_chain(block['height'] + 1, new_chain_id, False)
 
+    def store_election(self, height, election):
```

Comment:
The doc string says `Store election results` but the function name says `store_election`. It would be better to name the function too as `store_election_results`

Reply:
done

Reply:
I still see the function name as `store_election`